pkg/aws/client.go

```diff
@@ -1021,7 +1037,13 @@ func (c *awsClient) CreateSecretInSecretsManager(name string, secret string) (st
 }
 
 func (c *awsClient) DeleteSecretInSecretsManager(secretArn string) error {
-	_, err := c.smClient.DeleteSecret(
+	_, err := c.smClient.DescribeSecret(&secretsmanager.DescribeSecretInput{
```

Comment:
Are we aiming to determine if the `arn` is valid here?
If so, can we use the `arn.Parse` function and save an API call to AWS?

Reply:
No, this aims at checking if secret exists in aws account